sources/platform/actors/development/actor_definition/input_schema/specification.md

```diff
@@ -11,20 +11,17 @@ slug: /actors/development/actor-definition/input-schema/specification/v1
 
 ---
 
-The input schema serves two primary purposes for an Actor:
+The Actor input schema serves two primary purposes:
 
 - It defines the structure and validation rules for the input data the Actor accepts.
 - It determines the user interface components rendered in the Apify Console for configuring the Actor's input.
 
-By defining an input schema, you can provide a user-friendly interface for configuring your Actor while ensuring that the input data supplied by users adheres to the specified requirements and constraints.
+By defining an input schema, you can provide a user-friendly interface for configuring your Actor while ensuring that the input data supplied by users adheres to the specified requirements and constraints. Also, the input schema make it easy to call and integrate your Actor from external systems.
 
-You can specify input schema for an Actor in multiple ways:
+To define an input schema for an Actor, set `input` field in the `.actor/actor.json` file to an input schema object as described below, or path to a JSON file containing the input schema.
+For backwards compability, if the `input` field is omitted, the system looks for an `INPUT_SCHEMA.json` file in the `.actor` directory or for an `INPUT_SCHEMA.json` file in the Actor's root directory - but do not depend on this behavior as it might be removed in the future. 
```

Comment:
```suggestion
To define an input schema for an Actor, set `input` field in the `.actor/actor.json` file to an input schema object as described below, or path to a JSON file containing the input schema.

For backwards compability, if the `input` field is omitted, the system checks these locations:

  1. `INPUT_SCHEMA.json` in the `.actor` directory
  2. `INPUT_SCHEMA.json` in the Actor's root directory
 ```
 
 I would omit any information about what might or might not happen, if it does then we can update the documentation with a note. If we want to steer users in one direction we can add that some info about preferred or recommended way of doing things

Reply:
IMO it's quite normal in docs to say that some behavior might be deprecetated in the future, so that people don't use it - this is exactly the case

Reply:
Really? I see it all the time in docs that some feature is deprecated and might be removed in the future. See https://www.google.com/search?q=deprecated+and+might+be+removed+in+the+future

Reply:
But none of the results are from official documentation but rather community resources. 

Furthermore
`deprecated & might be removed in the future` =/= `might be removed in the future`,
if the intention is that these two ways of defining input schema are deprecated and & will be removed in the future, then let's mark them explicitly as deprecated and then mentioning that they might be removed in the future is a-ok

Reply:
Oh right, fair enough, I'll mention it's deprecated